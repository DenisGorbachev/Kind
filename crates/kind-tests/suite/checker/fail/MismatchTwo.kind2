Main (a: U60): U60
Main a = Type