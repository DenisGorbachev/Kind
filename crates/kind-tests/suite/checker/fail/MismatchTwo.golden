   ERROR  Type mismatch

      * Got      : Type 
      * Expected : U60 

      * Context: 
      *   a : U60 

      /--[suite/checker/fail/MismatchTwo.kind2:2:10]
      |
    1 |    Main (a: U60): U60
    2 |    Main a = Type
      |             v---
      |             \Here!


